pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -105,6 +105,16 @@ func (c *Client) OperatorExistsInCluster(name, namespace string) bool {
 	return true
 }
 
+// OperatorVersionExistsInCluster checks if a given OperatorVersion object is installed on the current k8s cluster
+func (c *Client) OperatorVersionExistsInCluster(name, namespace string) bool {
```

Comment:
Tom, we already have [OperatorVersionsInstalled](https://github.com/kudobuilder/kudo/blob/main/pkg/kudoctl/util/kudo/kudo.go#L336)

Reply:
This more closely mirrors OperatorExistsInCluster, which provides a quick boolean for checking rather than providing the list of OVs that the client would have to parse through.

see:

https://github.com/kudobuilder/kudo/blob/main/pkg/kudoctl/util/kudo/kudo.go#L97-L106